src/main/java/pikappi/task/Task.java

```diff
@@ -0,0 +1,80 @@
+package pikappi.task;
```

Comment:
I like how there are separate packages within the pikappi package to distinguish between the different class functionalities.